### PR TITLE
DP-823 - split documentation into separate parts

### PR DESCRIPTION
**Pull request by haavardeide**

### CODE REVIEW

File: doc/datasets.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+```bash
+origo datasets ls
+origo datasets ls <datasetid>
+origo datasets ls <datasetid> <versionid> <editionid>
+```
+To start exploring the datasets in Origo you do not need to log in, but: based on the permissions set on each dataset you might get different lists
+
+*Note*: For the correct, up to date, schema definition, please see [metadata-api schema catalogue](https://github.oslo.kommune.no/origo-dataplatform/metadata-api/tree/master/schema) (for now this is ), the datasets below are for demonstration purposes
+
+To search for a specific dataset you can use the `--filter` option to search for only a dddsubset of datasets available
```
Comment (fredriv) [inline doc/datasets.md:35]: Typo `dddsubset`

File: doc/install.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+Install
+=====
+
+# Requirements
+Origo CLI requires the following to run:
+* `python 3.6`
+* `make` and build tools for installing python modules, see output from `make init` below to see any other requirements you must install in order to install Origo CLI
+
+In addition it is recommended to use a tool like `jq` to parse and make automated descions based on output.
```
Comment (simenheg) [inline doc/install.md:9]: Another tiny one: "descions" => "decisions".

File: doc/configuration.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+Configuration
+=====
+
+* [Authentication](#authentication)
+* [Environment](#environment)
+
+# Authentication
+All CRUD operations needs a authenticated user in order to be executed. Some `ls` operations will not require a authenticated user, but might return a subset of available resources compared to that of a logged in user.
```
Comment (simenheg) [inline doc/configuration.md:8]: "needs" => "need"
"a authenticated" => "an authenticated"

File: doc/datasets.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+# List all datasets
+To explore datasets in Origo you can use the following commands:
+```bash
+origo datasets ls
+origo datasets ls <datasetid>
+origo datasets ls <datasetid> <versionid> <editionid>
+```
+To start exploring the datasets in Origo you do not need to log in, but: based on the permissions set on each dataset you might get different lists
+
+*Note*: For the correct, up to date, schema definition, please see [metadata-api schema catalogue](https://github.oslo.kommune.no/origo-dataplatform/metadata-api/tree/master/schema) (for now this is ), the datasets below are for demonstration purposes
```
Comment (simenheg) [inline doc/datasets.md:33]: Something missing after "for now this is "?

File: doc/events.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+Events
+=====
+To see all operations available on events:
+```
+origo events -h
+origo event_streams -h
+```
+
+TOC:
+* [What is a event](#what-is-a-event)
```
Comment (simenheg) [inline doc/events.md:10]: "a event" => "an event" (throughout)

File: doc/install.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+# Install
+Origo CLI is not yet installable as a application via pip (coming soon), but must be downloaded from github in order to be set up
```
Comment (simenheg) [inline doc/install.md:12]: "a application" => "an application"

**Pull request merged**